astroquery/heasarc/tests/test_heasarc_remote.py

```diff
@@ -96,3 +100,19 @@ def test_query_region(self):
             skycoord_3C_273, mission=mission, radius="1 degree")
 
         assert len(table) == 63
+
+    def test_query_region_nohits(self):
+        """
+        Regression test for #2560: HEASARC returns a FITS file as a null result
+        """
+        heasarc = Heasarc()
+
+        with warnings.catch_warnings(record=True) as warn:
```

Comment:
why not use `pytest.warns` here?

Reply:
because I didn't remember it was a thing